dandiapi/urls.py

```diff
@@ -45,13 +45,13 @@
     .register(
         r'versions',
         VersionViewSet,
-        basename='version',
+        basename='dandiset-version',
```

Comment:
[DRF 3.15 changes](https://www.django-rest-framework.org/community/release-notes/#3150):
> Raise ImproperlyConfigured exception if `basename` is not unique [[#8438](https://github.com/encode/django-rest-framework/pull/8438)]

I think this is a sensible new name, since the viewsets are nested.